network-api/networkapi/wagtailpages/rss.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.contrib.syndication.views import Feed
+from django.utils.feedgenerator import Atom1Feed
+
+from .models import BlogPage
+
+
+class RSSFeed(Feed):
+    """
+    Blog page RSS feed, using the content:encoded serializer above
+    """
+
+    title = 'Mozilla Foundation Blog'
+    link = 'https://foundation.mozilla.org/blog/'
+    feed_url = 'https://foundation.mozilla.org/blog/rss/'
+    description = 'The Mozilla Foundation Blog'
+
+    def items(self):
+        return BlogPage.objects.live().public()[:settings.FEED_LIMIT]
```

Comment:
Is this going to catch all blog pages or just children of /blog? Because blog template get used in other areas of the site (like campaigns).

Reply:
all BlogPage objects, so I can rewrite this to `reverse('/blog').children().public().live()` instead.